integration-tests/src/otc.rs

```diff
@@ -0,0 +1,141 @@
+//                    :                     $$\   $$\                 $$\                    $$$$$$$\  $$\   $$\
+//                  !YJJ^                   $$ |  $$ |                $$ |                   $$  __$$\ $$ |  $$ |
+//                7B5. ~B5^                 $$ |  $$ |$$\   $$\  $$$$$$$ | $$$$$$\  $$$$$$\  $$ |  $$ |\$$\ $$  |
+//             .?B@G    ~@@P~               $$$$$$$$ |$$ |  $$ |$$  __$$ |$$  __$$\ \____$$\ $$ |  $$ | \$$$$  /
+//           :?#@@@Y    .&@@@P!.            $$  __$$ |$$ |  $$ |$$ /  $$ |$$ |  \__|$$$$$$$ |$$ |  $$ | $$  $$<
+//         ^?J^7P&@@!  .5@@#Y~!J!.          $$ |  $$ |$$ |  $$ |$$ |  $$ |$$ |     $$  __$$ |$$ |  $$ |$$  /\$$\
+//       ^JJ!.   :!J5^ ?5?^    ^?Y7.        $$ |  $$ |\$$$$$$$ |\$$$$$$$ |$$ |     \$$$$$$$ |$$$$$$$  |$$ /  $$ |
+//     ~PP: 7#B5!.         :?P#G: 7G?.      \__|  \__| \____$$ | \_______|\__|      \_______|\_______/ \__|  \__|
+//  .!P@G    7@@@#Y^    .!P@@@#.   ~@&J:              $$\   $$ |
+//  !&@@J    :&@@@@P.   !&@@@@5     #@@P.             \$$$$$$  |
+//   :J##:   Y@@&P!      :JB@@&~   ?@G!                \______/
+//     .?P!.?GY7:   .. .    ^?PP^:JP~
+//       .7Y7.  .!YGP^ ?BP?^   ^JJ^         This file is part of https://github.com/galacticcouncil/HydraDX-node
+//         .!Y7Y#@@#:   ?@@@G?JJ^           Built with <3 for decentralisation.
+//            !G@@@Y    .&@@&J:
+//              ^5@#.   7@#?.               Copyright (C) 2021-2023  Intergalactic, Limited (GIB).
+//                :5P^.?G7.                 SPDX-License-Identifier: Apache-2.0
+//                  :?Y!                    Licensed under the Apache License, Version 2.0 (the "License");
+//                                          you may not use this file except in compliance with the License.
+//                                          http://www.apache.org/licenses/LICENSE-2.0
+#![cfg(test)]
+use crate::polkadot_test_net::*;
+use frame_support::assert_ok;
+use orml_traits::NamedMultiReservableCurrency;
+use pallet_otc::NAMED_RESERVE_ID;
+use xcm_emulator::TestExt;
+
+#[test]
+fn place_order_should_work() {
+	TestNet::reset();
+	Hydra::execute_with(|| {
+		// Act
+		assert_ok!(hydradx_runtime::OTC::place_order(
+			hydradx_runtime::Origin::signed(ALICE.into()),
+			DAI,
+			HDX,
+			20 * UNITS,
+			100 * UNITS,
+			false,
+		));
+
+		// Assert
+		let order = hydradx_runtime::OTC::orders(0);
+		assert!(order.is_some());
+		assert_eq!(
```

Comment:
small thing but for this I like using methods like:

assert_alice_native_named_reserve_balance

Or making it more generic with the name as param:

assert_native_named_reserve_balance

So the the domain becomes more explicit.